The 6 `ExpectError` regexes in `internal/fcs/cloud_azure_tenant_resource_test.go` no longer match the diagnostic, so the modified/new acceptance tests fail at plan time.

The lengthened validation message now wraps across two lines when Terraform renders it:

```
agentless_scanning_subscription_ids requires dspm or vulnerability_scanning
to be enabled.
```

The regex has a literal space between `vulnerability_scanning` and `to`, but the rendered output has a newline + indentation there, so it never matches. Running it:

```
$ TF_ACC=1 go test ./internal/fcs/ -run TestAccCloudAzureTenant_agentlessSubIdsRequiresDSPMOrVulnScanning -v
    cloud_azure_tenant_resource_test.go:480: Step 1/1, expected an error with pattern, no match on: ...
        agentless_scanning_subscription_ids requires dspm or vulnerability_scanning
        to be enabled.
--- FAIL: TestAccCloudAzureTenant_agentlessSubIdsRequiresDSPMOrVulnScanning
```

This affects 6 `ExpectError` lines (486, 512, 516, 542, 568, 594), across `_agentlessSubIdsRequiresDSPMOrVulnScanning`, `_agentless_UnknownDSPMEnabled`, `_agentless_UnknownSubIds`, `_agentless_UnknownDSPMObject`, and the new `_agentless_UnknownVulnerabilityScanningEnabled`. The validation logic itself is correct; only the test assertions are broken by the line wrap.

Suggested fix: switch to a raw-string regex and relax the wrap point with `\s+`:

```go
ExpectError: regexp.MustCompile(`agentless_scanning_subscription_ids requires dspm or vulnerability_scanning\s+to be enabled`),
```

Verified locally: with this change the full `internal/fcs` acceptance suite passes (all previously-failing Azure tests plus the new vulnerability_scanning coverage).

--- a/internal/fcs/cloud_azure_tenant_resource_test.go
+++ b/internal/fcs/cloud_azure_tenant_resource_test.go
@@ -483,7 +483,7 @@ func TestAccCloudAzureTenant_agentlessSubIdsRequiresDSPMOrVulnScanning(t *testin
 		Steps: []resource.TestStep{
 			{
 				Config:      testAccCloudAzureTenantConfig_agentlessSubIdsNoDSPM(tenantID, subID1),
-				ExpectError: regexp.MustCompile("agentless_scanning_subscription_ids requires dspm or vulnerability_scanning to be enabled"),
+				ExpectError: regexp.MustCompile(`agentless_scanning_subscription_ids requires dspm or vulnerability_scanning\s+to be enabled`),
 			},
 		},
 	})
@@ -509,11 +509,11 @@ func TestAccCloudAzureTenant_agentless_UnknownDSPMEnabled(t *testing.T) {
 			},
 			{
 				Config:      testAccCloudAzureTenantConfig_agentlessSubIdsUnknownDSPM(tenantID, subID1, false),
-				ExpectError: regexp.MustCompile("agentless_scanning_subscription_ids requires dspm or vulnerability_scanning to be enabled"),
+				ExpectError: regexp.MustCompile(`agentless_scanning_subscription_ids requires dspm or vulnerability_scanning\s+to be enabled`),
 			},
 			{
 				Config:      testAccCloudAzureTenantConfig_agentlessSubIdsNoDSPM(tenantID, subID1),
-				ExpectError: regexp.MustCompile("agentless_scanning_subscription_ids requires dspm or vulnerability_scanning to be enabled"),
+				ExpectError: regexp.MustCompile(`agentless_scanning_subscription_ids requires dspm or vulnerability_scanning\s+to be enabled`),
 			},
 		},
 	})
@@ -539,7 +539,7 @@ func TestAccCloudAzureTenant_agentless_UnknownSubIds(t *testing.T) {
 			},
 			{
 				Config:      testAccCloudAzureTenantConfig_unknownAgentlessSubIds(tenantID, subID1, false),
-				ExpectError: regexp.MustCompile("agentless_scanning_subscription_ids requires dspm or vulnerability_scanning to be enabled"),
+				ExpectError: regexp.MustCompile(`agentless_scanning_subscription_ids requires dspm or vulnerability_scanning\s+to be enabled`),
 			},
 		},
 	})
@@ -565,7 +565,7 @@ func TestAccCloudAzureTenant_agentless_UnknownDSPMObject(t *testing.T) {
 			},
 			{
 				Config:      testAccCloudAzureTenantConfig_unknownDSPMObject(tenantID, subID1, false),
-				ExpectError: regexp.MustCompile("agentless_scanning_subscription_ids requires dspm or vulnerability_scanning to be enabled"),
+				ExpectError: regexp.MustCompile(`agentless_scanning_subscription_ids requires dspm or vulnerability_scanning\s+to be enabled`),
 			},
 		},
 	})
@@ -591,7 +591,7 @@ func TestAccCloudAzureTenant_agentless_UnknownVulnerabilityScanningEnabled(t *te
 			},
 			{
 				Config:      testAccCloudAzureTenantConfig_agentlessSubIdsUnknownVulnScanning(tenantID, subID1, false),
-				ExpectError: regexp.MustCompile("agentless_scanning_subscription_ids requires dspm or vulnerability_scanning to be enabled"),
+				ExpectError: regexp.MustCompile(`agentless_scanning_subscription_ids requires dspm or vulnerability_scanning\s+to be enabled`),
 			},
 		},
 	})

